### PR TITLE
Data privacy (DEV)

### PR DESCRIPTION
**Pull request by harambasicluka**

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by mtwalli — DISMISSED** [review]

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 1]
```diff
@@ -544,7 +543,7 @@ <h3>
 <p>
     Der Fehlerbericht enthält jedoch keine Informationen über QR-Codes für die Testregistrierung,
     das in Ihrer App gespeicherte Token (siehe dazu unten Punkt 6 b. unter „Testergebnis abrufen“),
-    Schnelltest-Ergebnisse, digitale Covid-19-Impf- und Testzertifikate und Einträge in Ihrem
+    Schnelltest-Ergebnisse, digitale Covid-Impf- und Testzertifikate und Einträge in Ihrem
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/assets/privacy_de.html:546]: Please spell COVID with all letters capitalized: "...digitale COVID-Impf- und Testzertifikate..."

**Review by ChristinaTsigganos — CHANGES_REQUESTED** [review]
Please spell COVID with all letters capitalized, then this PR is approved by UA. Thanks a lot!

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Review by mtwalli — APPROVED** [review]

**Review by axelherbstreith — APPROVED** [review]

**Pull request merged**